addons/web/static/src/legacy/scss/bootstrap_overridden.scss

```diff
@@ -261,3 +261,6 @@ $form-switch-checked-bg-image: url("data:image/svg+xml,<svg xmlns='http://www.w3
 $badge-color: inherit !default;
 $badge-padding-y: 0.25em !default;
 $badge-padding-x: 0.4em !default;
+
+// Placeholder color 
+$input-placeholder-color: $gray-300 !default;
```

Comment:
This affects all input placeholders, so to see with @stefanorigano whether this is ok

Reply:
Thanks @alexkuhn .
@Brieuc-brd , I missed this notification. `$gray-300` is sensible lower than default (600)... have you tested the entire UI?

Reply:
@stefanorigano Yes, this variable seems to be used only for inputs with the `form-control` class. 
https://github.com/odoo/odoo/blob/d108db36ca2974179ded929370ea45a106f2e08c/addons/web/static/lib/bootstrap/scss/forms/_form-control.scss#L57

The purpose of this change is to have the same placeholder color with or without this class to keep consistency.
eg. input with placeholder without the `form-control` class (Project app):
![Capture d’écran 2022-07-29 à 14 45 01](https://user-images.githubusercontent.com/80679690/181764221-fa50374e-93c9-43f0-b383-d5bcf963245d.png)